src/HotChocolate/Fusion-vnext/src/Fusion.Execution/Execution/Nodes/OperationExecutionNode.cs

```diff
@@ -117,41 +117,49 @@ protected override async ValueTask<ExecutionStatus> OnExecuteAsync(
         }
         catch (Exception exception)
         {
-            AddErrors(context, exception, variables, _responseNames);
+            AddErrors(context, exception, variables);
             return ExecutionStatus.Failed;
         }
 
         var index = 0;
         var bufferLength = Math.Max(variables.Length, 1);
-        var buffer = ArrayPool<SourceSchemaResult>.Shared.Rent(bufferLength);
+        var resultBuffer = ArrayPool<SourceSchemaResult>.Shared.Rent(bufferLength);
+        var errorBuffer = ArrayPool<SourceSchemaErrors?>.Shared.Rent(bufferLength);
```

Comment:
Additional renting just to set the extension, since I didn't want to bleed the execution nodes into the source schema result parsing... have to revisit...